src/onboarding.md

```diff
@@ -0,0 +1,53 @@
+# Onboarding to the Project
+
+This document is a starting point for a new team member (or a refresh for
+existing team members) on things that are useful to know as a member of the
+project, in particular in terms of where to raise concerns or get help.
+
+## Relationship to Council
+
+The Leadership Council formally takes positions on behalf of the Project when
+needed. All Rust project members are officially represented by approximately
+one member of the council (some teams parent up to two or more members).
+
+Generally speaking, any concerns you have that are either more widely
+applicable (i.e., not just your team) or you feel aren't being fully handled
+within your team can be escalated to the Council. Note that interpersonal
+concerns and/or code of conduct violations should always be directed to the
+Moderation team.
+
+Escalating to the Council can be done via:
+
+- Zulip, on [#council](https://rust-lang.zulipchat.com/#narrow/channel/392734-council).
+- Zulip DM, to your council representative (see
+  [council](https://www.rust-lang.org/governance/teams/leadership-council) for
+  which team each representative represents).
+
+Either can be a reasonable starting point, please select what you feel more
+comfortable with, though bias towards being public.
+
+Leadership Council positions are elected by the Project teams on a rotating
+calendar; see [Council term limits] for more details.
+
+[Council term limits]: https://rust-lang.github.io/rfcs/3392-leadership-council.html#term-limits
```

Comment:
When possible, I'd like to keep links to the "official" policy pages, instead of RFCs which are more of a historical snapshot of a decision.

```suggestion
[Council term limits]: governance/council.md#term-limits
```